cmd/cmdutil/helpers.go

```diff
@@ -83,6 +94,21 @@ func ValidateArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func ValidateIntOrPercantageValue(value string) bool {
+	valid := false
+	if len(value) >= 1 && len(value) <= 7 {
```

Comment:
What happens if we give a percentage value that's >100%? Should we check if it's a percentage first, and then verify the value is between 1-100%?